corehq/apps/events/forms.py

```diff
@@ -134,3 +137,34 @@ def get_attendee_choices(self):
             (attendee.case_id, attendee.name)
             for attendee in AttendeeCase.objects.by_domain(self.domain)
         ]
+
+
+class NewAttendeeForm(forms.Form):
+    name = forms.CharField(
+        max_length=255,
+        required=True,
+        label=gettext_noop('Name'),
+    )
+
+    # TODO: Offer external_id?
+    #       Support uniqueness validation like NewMobileWorkerForm.username
+    # external_id = forms.CharField(
```

Comment:
Is this commented out intentionally? I know you mentioned figuring out a way to provide a unique id if two attendees have the same name. Is it going to be done using this `external_id`?

Reply:
Also to that (and definitely not necessary to discuss here, we can and will probably discuss this at another time closer to implementation), but I presume `external_id` is not necessarily a UUID string, right? I just think that whatever this value will be, it needs to mean something to the user in order to know which attendee is exactly which person.

Reply:
> (Weird that this thought came from two people named "C. Smit" )

Haha! Brilliant!!